guides/common/modules/proc_enabling-puppet.adoc

```diff
@@ -0,0 +1,54 @@
+:_module-type: PROCEDURE
+
+[id="proc_enabling-puppet_{context}"]
+= Enabling Puppet Integration with {Project}
+
+[role="_abstract"]
```

Comment:
What does this mean/do? I have only seen this in `guides/common/modules/proc_synchronizing-ansible-collections.adoc` beforce.  I don't think we need this/rely on this. Do you need this for downstream?

Same goes for line one.

If you don't know for sure why, then please remove both instances.